worker-core/src/main/java/com/hpe/caf/worker/core/WorkerCore.java

```diff
@@ -269,80 +326,88 @@ private boolean performJobStatusCheck(TaskMessage tm) throws InvalidJobTaskIdExc
             String statusCheckUrl = tracking.getStatusCheckUrl();
             if (statusCheckUrl == null) {
                 //If statusCheckUrl is null then we can't perform the status check so we have to assume the job is active.
-                return true;
+                return JobStatus.Active;
             }
 
             String jobId = tracking.getJobId();
             LOG.debug("Task {} (job {}) - attempting to check job status", tm.getTaskId(), jobId);
-            JobStatusResponse jobStatus = getJobStatus(jobId, statusCheckUrl);
-            long newStatusCheckTime = System.currentTimeMillis() + jobStatus.getStatusCheckIntervalMillis();
+            JobStatusResponse jobStatusResponse = getJobStatus(jobId, statusCheckUrl);
+            long newStatusCheckTime = System.currentTimeMillis() + jobStatusResponse.getStatusCheckIntervalMillis();
```

Comment:
Let's change the way the status check time is passed between the workers.  Instead of summing these and passing the sum let's pass them individually and allow the receiving worker to decide whether it should lookup the status or not.  For workers that have a paused queue I imagine they would want to lookup more frequently than the rest.

Reply:
Added in this commit: https://github.com/WorkerFramework/worker-framework/pull/125/commits/07ddc671ae4404f751efe8497982d7ba171d6243